source/includes/api/_before_you_get_started.md

```diff
@@ -56,6 +56,31 @@ Below you can find a list of RW API resources that use the `application` field:
 * [Vocabulary](/index-rw.html#vocabulary-and-tags)
 * [Widgets](/index-rw.html#widget10)
 
+## Caching
+
+Some services in the RW API rely on caching as a way to improve the performance and efficiency of their endpoints. By relying on caching mechanisms, services can serve pre-computed responses for commonly requested endpoints, reducing the need to perform the computations needed to serve the same request several times. Caching also has the added benefits of improving performance and reducing service load, which leads to better scalability overall.
```

Comment:
Caching should not be though of as a way to achieve improved performance. Caching should be primarily a scaling tool. Relying on caching for performance is acknowledging we are ok with sacrificing performance for a substantial part of our requests, especially the custom ones.